### PR TITLE
CI/CD for pypy packages using github actions

### LINKED ISSUE #10: [Feature Request] pip package CD
### Requirements
GitHub actions is needed on master to publish this package to `pypy` as soon as a PR is merged. Maintainers need to be subsequently notified about the same. 

### Potential notification media
*Any one*
- Telegram
- Mail
- Issue comment

### Additional features
GitHub actions for automatic release build with the `GIT_SHA` of the latest commit as version prefix.[This](https://github.com/c4milo/github-release) might help with the releases.

### PR DESCRIPTION
**Pull request by SphericalKat**
Preparation for distribution and packaging. Closes #10

### CODE REVIEW

File: setup.py
Comment (Geek-ubaid) [inline setup.py:16]: You can remove setuptools and wheel from here because you are already installing in actions.

**Review by Geek-ubaid — CHANGES_REQUESTED** [review]

**Review by Geek-ubaid — APPROVED** [review]

**Comment by L04DB4L4NC3R** [comment]
@Geek-ubaid don't merge right now. We have to add the login credentials of pypi as secrets.

**Comment by L04DB4L4NC3R** [comment]
> @Geek-ubaid don't merge right now. We have to add the login credentials of pypi as secrets.

The following secrets were added. It can be merged now.

```
PYPI_USERNAME
PYPI_PASSWORD
```

**Review by L04DB4L4NC3R — APPROVED** [review]

**Pull request merged**
